Makefile

```diff
@@ -13,6 +13,7 @@ endif
 .PHONY: tests
 tests:
 	(cd build && ctest)
+	@echo "-> NGS man is available at: " && man -w ngs1
```

Comment:
Let's not mix tests that can run after build but without installation (`ctest`) with ones that test the installation. Then, only run installation tests where appropriate.

Reply:
So in what place do you advice to add this tests?

Reply:
How about new target? Let's say `test-installation`?

Reply:
Added the test-installation as requested

Reply:
@ilyash should we add the man testing into our actions that test ngs build ?

Reply:
@organom , yes, where applicable